HACKING.md

```diff
@@ -41,6 +35,61 @@ release.
 
 We generally cut a release before disruptive changes land.
 
+### Building on Non-Linux Systems
+
+We provide the `openshift/release` container in which all of our build
+dependencies live, so that one can build a full release of OpenShift without
+having to install anything other than a container runtime on their local system.
+To run scripts or `make` targets from the Origin repo inside of the container,
+use:
+
+    $ hack/env ${COMMAND}
+
+For instance, to build the `oc` binary:
+
+    $ hack/env make build WHAT=cmd/oc
+
+The release container works by streaming a copy of the repository into a volume,
+sharing that volume with the container as it's working directory, executing the
+desired command and streaming the results back to your filesystem.
+
+One can configure what files are uploaded to the container by setting the
+`$OS_BUILD_ENV_EXCLUDE` variable. By default, the entire repository is streamed
+into the container except for anything under `_output/`.
+
+Similarly, to configure what files are downloaded from the container after the
+action is taken, use `$OS_BUILD_ENV_PRESERVE`. By default, `_output/local/bin`,
+`_output/local/releases`, and `_output/scripts` will be downloaded.
+
+### Building Individual Binaries
+
+While `make release` and `make build` will build all of the binaries required
+for a full release of OpenShift, it is also possible to build individual
+binaries. Binary entrypoints are kept under the `cmd/` directory and can be
+specified to build with the `WHAT` parameter, for instance, to build just `oc`:
+
+    $ make build WHAT=cmd/oc
+
+### Building Individual Images
+
+The `make release` and `hack/build-images.sh` scripts will build the entire
```

Comment:
i thought you said just make release did it?

hack/build-images.sh doesn't have much value on its own at this point since you need to build new rpms first.